Pod/Classes/WPNavigationMediaPickerViewController.m

```diff
@@ -382,6 +386,7 @@ - (void)navigationController:(UINavigationController *)navigationController will
 - (void)showAfterViewController:(UIViewController *)viewController
 {
     NSParameterAssert(viewController);
+    self.afterSelectionViewController = viewController;
```

Comment:
It is not very clear to me what an `afterSelectionViewController` is, can you explain a bit?

Reply:
So this method was something that was added to WPMediaPicker to cater to the scenarios that you want to show a VC in the same navigation controller of the picker after a selection is made. This was added to cater for the Crop option that we see in the main apps. 

Before this was available we were closing the picker and open another screen in modal, but the UI felt very bad, so we implemented this.

I agree with you that this name is not great, do you have a suggestion for a better name?

Reply:
OK thanks, I don't have any problem with naming, I was just trying to understand how this is solving. 👍